RatScanner/View/BlazorOverlay.xaml.cs

```diff
@@ -15,43 +18,86 @@ namespace RatScanner.View;
 /// </summary>
 public partial class BlazorOverlay : Window
 {
+	private IntPtr overlayHandle;
 	public BlazorOverlay(ServiceProvider serviceProvider)
 	{
 		Resources.Add("services", serviceProvider);
 
+		overlayHandle = new WindowInteropHelper(this).Handle;
+		// Set the  overlay window style such that alt+tab does not show it in the switcher
+		SetWindowLong(overlayHandle, GWL_EX_STYLE, (GetWindowLong(overlayHandle, GWL_EX_STYLE) | WS_EX_TOOLWINDOW) & ~WS_EX_APPWINDOW);
+
 		InitializeComponent();
 	}
 
 	private void BlazorOverlay_Loaded(object sender, RoutedEventArgs e)
 	{
 		blazorOverlayWebView.WebView.DefaultBackgroundColor = System.Drawing.Color.Transparent;
-		SetSize();
+
+		// Set up the window check timer
+		initializeWindowCheckTimer();
+
+		// Finish loading the webview
 		blazorOverlayWebView.WebView.NavigationCompleted += WebView_Loaded;
 		blazorOverlayWebView.WebView.CoreWebView2InitializationCompleted += CoreWebView_Loaded;
+
+		// Other customizations
+		this.IsHitTestVisible = false;
 	}
+	
+	private void initializeWindowCheckTimer()
+	{
+		// Set up the timer with interval
+		windowCheckTimer = new System.Timers.Timer(10000);
+		windowCheckTimer.Elapsed += checkWindowEvent;
+		windowCheckTimer.AutoReset = true;
+		windowCheckTimer.Enabled = true;
 
-	[DllImport("user32.dll", SetLastError = true)]
-	static extern bool SetWindowPos(IntPtr hWnd, IntPtr hWndInsertAfter, int X, int Y, int cx, int cy, UInt32 uFlags);
+		// Trigger the timer immediately at initialization
+		checkWindowEvent(null, null);
+	}
 
-	private void SetSize()
+	private void checkWindowEvent(Object source, ElapsedEventArgs e)
 	{
-		var bounds = Screen.AllScreens.Select(screen => screen.Bounds);
-		var left = 0;
-		var top = 0;
-		var right = 0;
-		var bottom = 0;
-		foreach (var bound in bounds)
+		// Used to find the area of the EFT game client to set up the overlay location and size
+		try
 		{
-			if (bound.Left < left) left = bound.Left;
-			if (bound.Top < top) top = bound.Top;
-			if (bound.Right > right) right = bound.Right;
-			if (bound.Bottom > bottom) bottom = bound.Bottom;
+			ScreenScale? gameScreenScale = RatScannerMain.Instance?.GameScreenScale;
+			// Set the size of the overlay now that we have the rect of EFT
+			//SetSize(rect.Left, rect.Top, rect.Width, rect.Height);
+			if (gameScreenScale != null)
+			{
+				Rectangle gameRect = GameWindowLocator.GetWindowLocation();
+				Dispatcher.Invoke(() =>
```

Comment:
Part of our timer which tells us where to move the overlay when we detect a game window